src/fetchers/base_fetcher.py

```
@@ -192,6 +192,34 @@ def fetch_csv_sentinel_weekly_age(
+        start_sub_period: str = "1",
+        end_year: str = "2025",
+        end_sub_period: str = "1",
+        pref_code: str = "13",
+        hc_code: str = "00",
+        epid_code: str = "0000",
+        total_mode: str = "0",
+    ) -> bytes:
+        """
+        定点監視 週報告分 特殊報告 性別集計表CSVを取得する
+        """
+        # 特殊報告用のエンドポイントIDを使用（仮に "9" とする）
```

Ruff (0.14.2)

Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)